### PR TITLE
Even cm

### PR DESCRIPTION
**Pull request by morecallan**
@katerebekah @guthb

### CODE REVIEW

File: Mathmagician.Tests/Mathmagician.Tests.csproj [hunk 1]
```diff
@@ -57,7 +57,11 @@
     <Compile Include="UnitTest1.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
     <Compile Include="IntegerTests.cs" />
+<<<<<<< HEAD
```
Comment (katerebekah) [inline Mathmagician.Tests/Mathmagician.Tests.csproj]: Is this from a merge conflict? If so, does this build with the <<<<<<HEAD in it?

File: Mathmagician/Even.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+{
+    public class Even
+    {
+
+        public List<int> ReturnEvenNumbersList(int userInput)
+        {
+            List<int> my_evens = new List<int>();
+
+            for (int i = 0; i < userInput; i++)
+            {
```
Comment (katerebekah) [inline Mathmagician/Even.cs:17]: Is 0 an even number?

File: Mathmagician/Mathmagician.csproj [hunk 3]
```diff
@@ -44,7 +44,11 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
+<<<<<<< HEAD
```
Comment (katerebekah) [inline Mathmagician/Mathmagician.csproj]: Again, does this compile?

File: Mathmagician/Even.cs (hunk 2 above)
Comment (morecallan) [inline Mathmagician/Even.cs:17]: @katerebekah apparently! I didn't think so at first, either.

**Comment by morecallan** [comment]
@katerebekah thanks for the feedback. I had noticed those issues and committed the changes but forgot to push that part up! I also noticed it was building even though it had the HEAD in program.cs as well and fixed that.

**Comment by katerebekah** [comment]
:+1:

**Comment by guthb** [comment]
👍

**Pull request merged**